File: CArticulo/CArticulo/Articulo.cs
using System;
namespace CArticulo
{
    public class Articulo
    {
        public Articulo()
        {
        }

        private long categoria;
        private long id;
        private string nombre = "";
        private decimal precio;

        public long Categoria{
            get { return categoria; }
            set { categoria = value; }
        }

        public long Id{
            get { return id; }
            set { id = value; }
        }

        public string Nombre{
            get { return nombre; }
            set { nombre = value; }
        }

        public decimal Precio{
            get { return precio; }
            set { precio = value; }
        }
    }
}

File: GArticulo/src/main/java/serpis/ad/Articulo.java
package serpis.ad;

import java.math.BigDecimal;

public class Articulo {
	private long id;
	private String nombre;
	private BigDecimal precio;
	private long categoria;
	
	public void setId(long id) {
		this.id = id;
	}
	
	public void setNombre(String nombre) {
		this.nombre = nombre;
	}
	
	public void setPrecio(BigDecimal precio) {
		this.precio = precio;
	}
	
	public void setCategoria(long categoria) {
		this.categoria = categoria;
	}
	
	public long getId() {
		return id;
	}
	
	public String getNombre() {
		return nombre;
	}
	
	public BigDecimal getPrecio() {
		return precio;
	}
	
	public long getCategoria() {
		return categoria;
	}
	
	
	

}
